src/utils/formatPriceToTwoDecimals.ts

```diff
@@ -0,0 +1,3 @@
+export function formatToTwoDecimals(value: number): string {
+  return value.toFixed(2)
```

Comment:
So this will actually truncate the number to decimals whereas I think you would rather round it, 
eg with this code 10.97777 would become 10.97 but it should rather be 10.98

Reply:
for some reason using rounding is cutting the decimals to all be .00, instead of their original decimal value, not sure if its the best idea

Reply:
Just double check the brackets in your rounding function, I also had an issue 😄

Reply:
think it's sorted now